doc/contributor/reference/architecture/serving-data.md

```diff
@@ -0,0 +1,15 @@
+# Serving the QGreenland core zip package
+
+The QGreenlnad core zip package is served to the public via an NGINX server that
+is run on NSIDC's virtual machine infrastructure.
+
+For details on how nginx is configured, see:
+
+* {github}`nginx`
+* {github}`docker-compose.yml`
```

Comment:
I've not seen this syntax before. Is this new or have we used it elsewhere?

Reply:
We have used this in a few other places (e.g., in `configuration.md`). This is made possible via the `extlinks` setting in `conf.py`.

Reply:
Ahhh. Thanks for the refresher!